tests/rptest/services/honey_badger.py

```diff
@@ -0,0 +1,42 @@
+# Copyright 2021 Vectorized, Inc.
+#
+# Use of this software is governed by the Business Source License
+# included in the file licenses/BSL.md
+#
+# As of the Change Date specified in that file, in accordance with
+# the Business Source License, use of this software will be governed
+# by the Apache License, Version 2.0
+import random
+import requests
+
+
+class HoneyBadger:
```

Comment:
you might consider adding the raw requests to admin.py. then having a hb specific interface on top in a class like this.